Assets/MixedRealityToolkit.Tests/PlayModeTests/TestFixture_02_ManipulationTests.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if !WINDOWS_UWP
+// When the .NET scripting backend is enabled and C# projects are built
+// Unity doesn't include the the required assemblies (i.e. the ones below).
+// Given that the .NET backend is deprecated by Unity at this point it's we have
+// to work around this on our end.
+using Microsoft.MixedReality.Toolkit.UI;
+using NUnit.Framework;
+using System.Collections;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.TestTools;
+using Microsoft.MixedReality.Toolkit;
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    public class TestFixture_02_ManipulationTests
+    {
+        /// <summary>
+        /// Test creating adding a ManipulationHandler to GameObject programmatically.
+        /// Should be able to run scene without getting any exceptions.
+        /// </summary>
+        /// <returns></returns>
+        [UnityTest]
+        public IEnumerator Test01_ManipulationHandlerInstantiate()
+        {
+            TestUtilities.InitializeMixedRealityToolkitScene(true);
+
+            var playspace = MixedRealityToolkit.Instance.MixedRealityPlayspace;
```

Comment:
This playspace setup is common to a number of they playmode tests, and might be worth having a TestUtilities.InitializePlayspace().

Note that it's possible to go either way here - having code written out this way has an advantage in that the entire test is clearly readable (in terms of how it is setting up the playspace, and then where the cube gets put, etc.

Reply:
At the same time if playspace creation (i.e. the light location + playspace lookat) will basically be the same for our playspace tests then having a single thing would probably reduce the noise in the tests to just what is being tested.

Reply:
lights are not always needed, but I did create an InitializePlayspace method.